src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -530,6 +532,7 @@ public void TestExecuteNonQuery(string connection, bool isAsync)
                         transaction: null,
                         columnEncryptionSetting: SqlCommandColumnEncryptionSetting.Enabled))
                     {
+                        sqlCommand.CommandTimeout = 90;
```

Comment:
This is a lot of timeout, any specific reason to add this?

Reply:
In some cases the none-localhost servers takes longer time to respond and that is usually more than default sqlcommand  commandTimeOut. we can decrease that. I think when the slcommand is disposed that time out is going with it as well. Just leaving enough room for time out.